fetch.bs

```diff
@@ -4303,14 +4332,19 @@ Range Requests</cite>. [[HTTP-RANGE]] However, this is not widely supported by b
     <var>httpRequest</var>'s
     <a for=request>credentials mode</a>.
 
+   <li>
+    <p>Let <var>httpCache</var> be the result of <a>determining the HTTP cache partition</a> for
+    <var>httpRequest</var>. If <var>httpCache</var> is null, fetch behaves as if there is no HTTP cache
+    at all and the <a>cache mode</a> is considered "<code>no-cache</code>".
```

Comment:
"fetch behaves as if there is no HTTP cache at all and the <a>cache mode</a> is considered "<code>no-cache</code>"." is not precise enough. You should add a second step which says something like

```html
If <var>httpCache</var> is null, then set <var>httpRequest</var>'s <a for=request>cache mode</a> to "<code>no-cache</code>".
```

Reply:
done